.gitignore

```diff
@@ -24,3 +24,4 @@
 *.ji
 
 .DS_Store
+get_toolchain.log
```

Comment:
I'm not opposed to adding this file to gitignore, it comes from some windows-specific setup scripts. It could maybe be a separate commit or PR but rebasing it out can be slightly complicated.